packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -218,9 +218,17 @@ export class FilesService {
         return withFilesSelect(this.selectQueryBuilder(), { imageId }).getSingleResult();
     }
 
-    async create({ folderId, ...data }: CreateFileInput & { copyOf?: FileInterface }): Promise<FileInterface> {
+    async create({ folderId, importSourceId, importSourceType, ...data }: CreateFileInput & { copyOf?: FileInterface }): Promise<FileInterface> {
```

Comment:
Why this special treatment? Spreading the fields using `...data` should work as well.